x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -116,14 +121,23 @@ const allSavedObjectTypes = [
 /**
  * Callbacks supported by the Fleet plugin
  */
-export type ExternalCallback = [
-  'packagePolicyCreate',
-  (
-    newPackagePolicy: NewPackagePolicy,
-    context: RequestHandlerContext,
-    request: KibanaRequest
-  ) => Promise<NewPackagePolicy>
-];
+export type ExternalCallback =
```

Comment:
It's outside the scope of this PR, but I think it's time to give these types names. Referencing `ExternalCallback[0]` and `ExternalCallback[1]` is atypical in our code and I think names `ExternalCallbackType` and `ExternalCallbackHandler` are preferable. I can open a ticket.